translations/harbour-countryquiz-sv.ts

```diff
@@ -554,7 +554,7 @@
         <location filename="../qml/components/StatsList.qml" line="71"/>
         <source>%1 / %2 for %3 p</source>
         <extracomment>%1 is number of right answers out of %2 total answer to reward %3 points, try to keep this short</extracomment>
-        <translatorcomment>Piste lopussa, koska suomessa lyhenteet loppuvat pisteeseen.</translatorcomment>
+        <translatorcomment>%1 är antal rätt svar av totalt %s svar för att bli belönad med %3 poäng. Försök att hålla detta kort.</translatorcomment>
```

Comment:
Btw, this is actually supposed to be additional comment from translator. You can have one or just skip it. In Finnish translation I told that there is a dot at the end because abbreviations end with a dot in Finnish (well, many do, there are rules).